packages/tabbed/src/index.js

```diff
@@ -0,0 +1,48 @@
+// vendor modules
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import getClassnames from 'classnames';
+// css modules
+import './index.scss';
+
+/**
+ * @example ../README.md
+ */
+export default class Tabbed extends React.Component {
+  static propTypes = {
+    isVertical: PropTypes.bool,
```

Comment:
Change "isVertical" to "layout" and default the value to "vertical" and the other option is "horizontal".

Because,
isVertical = false does not necessarily mean its horizontal, it can also mean diagonal.